corehq/apps/reports/export.py

```diff
@@ -89,7 +90,7 @@ def generate_bulk_file(self):
                                 separator=self.separator)
                         ])]
                     )
-
+                update_progress(i+1)
```

Comment:
Except that we don't know how many docs there are so we can't know what the
'total' is.